substrate/client/transaction-pool/src/graph/pool.rs

```diff
@@ -304,6 +305,7 @@ impl<B: ChainApi, L: EventHandler<B>> Pool<B, L> {
 		at: &HashAndNumber<B::Block>,
 		parent: <B::Block as BlockT>::Hash,
 		extrinsics: &[RawExtrinsicFor<B>],
+		inactive_views_xts_to_tags: Arc<HashMap<ExtrinsicHash<B>, Option<Vec<Tag>>>>,
```

Comment:
I'd prefer to have:
```
Arc<Option<HashMap<ExtrinsicHash<B>, Vec<Tag>>>>
```

Not sure why you proposed `Option<Vec<Tag>>>? If the tag is not known the tx should not be in the map.

Reply:
For old pool, `None` should be passed.

Reply:
> Not sure why you proposed `Option<Vec>>? If the tag is not known the tx should not be in the map

Addressed this here: [8de3686](https://github.com/paritytech/polkadot-sdk/pull/7980/commits/8de368694385db28c903323ad9779fc1b87b9dda).

> Arc<Option<HashMap<ExtrinsicHash<B>, Vec<Tag>>>>

Yup, makes sense, will change 👍